inc/command/class-command.php

```diff
@@ -487,6 +518,58 @@ protected function run_command( InputInterface $input, OutputInterface $output,
 		return $return_val;
 	}
 
+	/**
+	 * Bootstrap tests folder.
+	 *
+	 * @param string $tests_folder Target folder to boostrap the tests into.
+	 * @param InputInterface $input
+	 * @param OutputInterface $output
+	 *
+	 * @return void
+	 */
+	protected function bootstrap_codecept( string $tests_folder, InputInterface $input, OutputInterface $output ) {
+		$default_suites = $this->get_test_suites( 'vendor/altis/dev-tools/tests' );
+
+		$suites = $input->getArguments()['options'][1] ?? '';
+		if ( 0 === strpos( $suites, '-' ) ) {
```

Comment:
Still not sure where the dash would be coming from here or how reliable it is.

Reply:
The dash represents a parameter argument, and the only case we're handling here is something that's passed beyond the end of options signal, eg: `codecept bootstrap -- -x something`.
`codecept bootstrap -- -x something suite1,suite2` seems like quite an edge case to expect, also non-standard approach.